cli/entryPoint.sh

```diff
@@ -19,7 +19,18 @@ echo -e "
 echo -e ""
 
 export HOME=/opfab
-# Add opfab-cli to the prompt in red
-export PS1="\033[31mOPFAB-CLI\033[0m \w \$ "
+if [ -z "$ENV_NAME" ]; then
+    # Add opfab-cli to the prompt in red
+    export PS1="\033[32mOPFAB-CLI\033[0m \w \$ "
+else
+    export PS1="\033[32mOPFAB-CLI\033[0m - \033[31m$ENV_NAME\033[0m \w \$ "
+fi
+
+if [ -n "$OPFAB_URL" ]; then
+    opfab login $OPFAB_URL $OPFAB_PORT $OPFAB_LOGIN  $OPFAB_PASSWORD
+else
+    echo "No configuration provided. Please run 'opfab login' to login to OperatorFabric."
```

Comment:
This sentence needs the following correction:
Please run 'opfab login' to log in to OperatorFabric.